fe/be-java-extensions/java-common/src/main/java/org/apache/doris/common/jni/JniScanner.java

```diff
@@ -61,6 +65,11 @@ public VectorTable getTable() {
         return vectorTable;
     }
 
+    public String getTableSchema() throws IOException {
```

Comment:
If we return `Map<String, String>` directly, do we no longer need `TableSchema`.

Reply:
TableSchema is required. The complex types of avro include map, array, etc., which need to be nested.